frontend/src/utils/hotkey_listener.ts

```diff
@@ -251,18 +258,19 @@ export class HotkeyListener {
     return keys.replace(/\b(ctrl|control)\b/gi, 'command')
   }
 
-  getPlatform() : Platform | undefined {
-    const {
-      windows, linux, macos
-    } = platformDetect
-
-    if (windows) {
-      return 'win'
-    } else if (macos) {
-      return 'mac'
-    } else if (linux) {
-      return 'linux'
+  getPlatform() : Platform {
```

Comment:
Updated `getPlatform` method to use custom code instead of `platform-detect` library